backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -72,6 +73,14 @@ func (m *FakeBadObjectStore) GetFromYamlFile(o interface{}, filePath string) err
 	return util.NewInternalServerError(errors.New("Error"), "bad object store")
 }
 
+func (m *FakeBadObjectStore) GetSignedUrl(bucketConfig *objectstore.Config, secret *corev1.Secret, expirySeconds time.Duration, artifactURI string) (string, error) {
```

Comment:
they are required by `ObjectStoreInterface`

Reply:
I mean you can omit their names but leave the types. It's a valid syntax and you'll still implement the interface.

Reply:
But the syntax requires you to remove all of them or none.

Reply:
As per slack discussion, I misunderstood this. I have removed the redundant names for the new files added, but kept them for existing ones for consistency. It will need to be a separate task to refactor the existing mock clients hollistically.